src/keycode.h

```diff
@@ -34,16 +34,16 @@ class KeyPress
 	KeyPress();
 	KeyPress(const char *name);
 
-	KeyPress(const irr::SEvent::SKeyInput &in, bool prefer_character=false);
+	KeyPress(const irr::SEvent::SKeyInput &in, bool prefer_character = false);
```

Comment:
no, it's a choice of LLVM to have readable code when using equal, and i'm not afraid, i prefer having it it permit to view more easily default values